docs: vale DITA compliance for installing and multitenancy assemblies (#8323)
Apicurio/apicurio-registry: docs/modules/ROOT/pages/getting-started/assembly-implementing-multitenancy.adoc

include::{mod-loc}shared/all-attributes.adoc[]

// assembly-implementing-multitenancy
[id="implementing-multitenancy_{context}"]
:_mod-docs-content-type: ASSEMBLY
= Implementing multitenancy with {registry}

[role="_abstract"]
Implement multitenancy with {registry} on {kubernetes} using the {operator}.
{registry} uses a multi-instance operator pattern. Each tenant receives a dedicated
`ApicurioRegistry3` custom resource (CR), which the operator reconciles into an independent set of {kubernetes}

> resources. This approach provides strong isolation between tenants at the infrastructure level. It uses

Reviewer: Are split lines handled by the conversion process? If not, L13 and L12 can be combined with L10.

Author: Split lines are handled fine by the conversion process — it works at the section level, not line level. Combined them anyway for readability. Thanks!

* xref:multitenancy-lightweight-group-isolation_{context}[]

.Prerequisites

* {installing-the-registry-openshift}

> * Ensure that you have installed the {operator}. See xref:deploying-registry-operator_{context}[].

Reviewer: ```suggestion
* The {operator} is installed. See xref:deploying-registry-operator_{context}[].
```

Author: Applied, thanks. Updated the other prerequisites in the file to match.

└── tenant-beta namespace
    └── ApicurioRegistry3: registry
----
+
This pattern provides stronger isolation using {kubernetes} RBAC, ResourceQuotas, and NetworkPolicies at the
{kubernetes-namespace} level. It is the recommended approach for most production environments.

Pattern C: Operator per namespace::
Each {kubernetes-namespace} has its own operator instance. The instance is restricted to watching only that {kubernetes-namespace}

> by using the `APICURIO_OPERATOR_WATCHED_NAMESPACES` environment variable.

Reviewer: Same question re sentences split over multiple lines.

Author: Same as above — combined for readability, but the conversion handles them fine.
